src/terminal/adapter/DispatchTypes.hpp

```diff
@@ -395,6 +395,7 @@ namespace Microsoft::Console::VirtualTerminal::DispatchTypes
 
     enum ModeParams : VTInt
     {
+        IRM_InsertReplaceMode = ANSIStandardMode(4),

```

Comment:
I love that my weird "tagged enum" thing didn't turn into YAGNI. I added it _because_ we'd eventually do IRM, and need `SM`/`RM`. 😄